Ginger/Ginger/Activities/ActivitiesGroupPage.xaml.cs

```diff
@@ -76,7 +79,131 @@ public ActivitiesGroupPage(ActivitiesGroup activitiesGroup, BusinessFlow parentB
                 SharedRepoInstanceUC.Visibility = Visibility.Collapsed;
                 SharedRepoInstanceUC_Col.Width = new GridLength(0);
             }
-        }     
+        }
+
+        void UpdateSharedRepositorySupportedOperations()
+        {
+            grdGroupedActivities.ShowUpDown = Visibility.Visible;
+
+            grdGroupedActivities.AddToolbarTool(Amdocs.Ginger.Common.Enums.eImageType.Add, "Add Another Activity to this Group", BtnAdd_Click);
+            grdGroupedActivities.AddToolbarTool(Amdocs.Ginger.Common.Enums.eImageType.DeleteSingle, "Delete Current Selected Activity from this Group", GroupActivitiesHandler);
+        }
+
+        private void BtnDelete_Click(object sender, RoutedEventArgs e)
+        {
+            FrameworkElement fElem = sender as FrameworkElement;
```

Comment:
for what you need to handle BtnDelete_Click? delete ucGrid itm is generic

Reply:
Done !